scripts/stop-server

```diff
@@ -28,14 +28,15 @@ if os.path.exists("/etc/supervisor/conf.d/zulip/zulip_db.conf"):
 if os.path.exists("/etc/supervisor/conf.d/zulip/thumbor.conf"):
     services.append("zulip-thumbor")
 
-# Contrary to the order in (re)start-server, we stop django before the
-# workers, to increase the chance that we finish processing any work
-# that may have been enqueued by the Django, leaving the final state
-# closer to "empty."  We stop Django before Tornado so it doesn't try
-# to make requests to make queues with a down'd Tornado.
-services.append("zulip-django")
-services.extend(["zulip-tornado", "zulip-tornado:*"])
-services.append("zulip-workers:*")
+if os.path.exists("/etc/supervisor/conf.d/zulip/zulip.conf"):
+    # Contrary to the order in (re)start-server, we stop django before the
+    # workers, to increase the chance that we finish processing any work
+    # that may have been enqueued by the Django, leaving the final state
+    # closer to "empty."  We stop Django before Tornado so it doesn't try
+    # to make requests to make queues with a down'd Tornado.
+    services.append("zulip-django")
+    services.extend(["zulip-tornado", "zulip-tornado:*"])
+    services.append("zulip-workers:*")
 
 subprocess.check_call(["supervisorctl", "stop", *services])
```

Comment:
This means we could conceivably call `supervisorctl stop` with no arguments -- which has an exit code of 1, so we'd exit here.  Which I think is fine.

Reply:
At least for `restart-server`, we might want `if len(workers) > 0` guards on some of these blocks?  Not sure.

Reply:
@timabbott I implemented some checks in `restart-server` - what do you think?